gooey/application.py

```diff
@@ -0,0 +1,18 @@
+'''The command line interface.'''
+
+import sys
+from importlib import import_module
+
+from gooey import Gooey
+
+
+def main(args=None):
+    args = args or sys.argv[1:]
+    module_path, function_name = args[0].split(":")
+    if len(args) > 1:
+        if args[1] == "--":
+            del args[1]
+    sys.argv = ["gooey", *args[1:]]
```

Comment:
Need to change "gooey" to the actual script name, though not sure how to infer it from module path + function name yet.